Source/WebCore/platform/mediarecorder/cocoa/AudioSampleBufferCompressor.h

```diff
@@ -32,7 +32,8 @@
 #include <wtf/WorkQueue.h>
 
 typedef struct opaqueCMSampleBuffer *CMSampleBufferRef;
-typedef struct OpaqueAudioConverter* AudioConverterRef;
+typedef struct OpaqueAudioConverter *AudioConverterRef;
```

Comment:
This is technically incorrect; AudioConverterRef isn't an Obj-C type, so the indirection symbol goes next to the type.

Reply:
I thought it was the objC style and that was determined by the file language rather than the object.